app/src/components/domain/RiskImminentEvents/WfpAdam/EventDetails/LayerDetails/styles.module.css

```diff
@@ -0,0 +1,46 @@
+.content {
+    display: flex;
+    flex-direction: column;
+    gap: var(--go-ui-spacing-xs);
+    padding-left: var(--go-ui-spacing-lg);
+
+    svg {
+        padding: var(--go-ui-spacing-xs);
+        width: 24px;
+        height: 24px;
```

Comment:
These values seem to be reused often. Let's define CSS variables for each and then reuse them.